test/utils.test.js

```diff
@@ -70,4 +70,42 @@ describe('Utils', function () {
 		assert.ok(Utils.getValueFromJsVariable);
 	});
 
+	it('should provide sanitise functionality', function () {
+		[
+			{ param: '   with space  ', result: 'with space' },
+			{ param: 'noSpace', result: 'noSpace' }
+	 	].forEach(function (test) {
+			assert.equal(Utils.sanitise(test.param), test.result);
+		});
+	});
+
+	it('should provide assignIfUndefined functionality', function () {
+		[
+			{
+				subject: { a: 'aa', b: 'b', c: 'c' },
+				target: { a: 'a', b: undefined },
+				result: { a: 'a', b: 'b', c: 'c' }
+			}
+		].forEach(function (test) {
+			Utils.assignIfUndefined(test.subject, test.target);
+			assert.deepEqual(test.target, test.result);
+		});
+	});
+
+	it('should provide whitelistProps functionality', function () {
+		const list = [
+			'contentId',
+			'subtype',
+			'type',
+		];
+		const resultProps = { contentId: 1234, subtype: 'subtype' };
+		const testProps = Object.assign(
+			{},
+			resultProps,
+			{ type: undefined, name: 'name' }
+		);
```

Comment:
Tiny thing again but you could refactor this to match the array style in the rest of this spec?

Reply:
👍